R/dataProcess.R

```diff
@@ -175,7 +252,6 @@ dataProcess = function(
 #' 
 MSstatsSummarize = function(proteins_list, method, impute, censored_symbol,
```

Comment:
let's rename this now

Reply:
I'm concerned about the backwards compatibility of renaming this because `proteins_list` is a list of data frames whereas `input` (from MSstatsSummarizeV2) is a dataframe.   If this function wasn't exported, I'd be fine with renaming.

Reply:
But at the same time, I'm wondering if clients use MSstatsSummarize ever.  I don't see why we would want to export this function.

Reply:
it's useful for modularized processing. All MSstatsX functions should be exported

Reply:
Ok, in that case, renaming MSstatsSummarizeV2 to MSstatsSummarize and MSstatsSummarize to something else is not backwards compatible.

Reply:
Nvm saw the previous comment above